pkg/cross-compilers/Dockerfile

```diff
@@ -26,12 +26,18 @@ RUN tar -xzvf aports.tar.gz --strip-components=1 && rm -rf aports.tar.gz
 
 # Versions must be aligned with content of APKBUILD
 ENV BINUTILS_VERSION 2.38
+# v1.2.3:
+ENV MUSL_REVISION="7a43f6fea9081bdd53d8a11cef9e9fab0348c53d"
 ENV MUSL_VERSION 1.2.3
 ENV GCC_VERSION 11.2.1_git20220219
 ADD --chown=builder:abuild https://ftp.gnu.org/gnu/binutils/binutils-${BINUTILS_VERSION}.tar.xz /var/cache/distfiles/binutils-${BINUTILS_VERSION}.tar.xz
-ADD --chown=builder:abuild https://git.musl-libc.org/cgit/musl/snapshot/v${MUSL_VERSION}.tar.gz /var/cache/distfiles/musl-v${MUSL_VERSION}.tar.gz
+WORKDIR /usr/src
+ADD https://github.com/bminor/musl.git#${MUSL_REVISION} /usr/src/${MUSL_VERSION}
```

Comment:
@christoph-zededa , I need to ask... is this a reliable official mirror? I'm not wiling to use non-official mirrors... but we might open an exception depending on each case....

Reply:
It looks like there is no official mirror, one alternative is to calculate sha256 from the official version and do the checking in the Dockerfile.... TBH I think is only worth if we are experiencing the issue with the official tarball a lot....